pkg/apimachinery/discoverysummarizer/types.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package discoverysummarizer
+
+// List of servers from which group versions should be summarized.
+type FederatedServerList struct {
+	Servers []FederatedServer `json:"servers"`
+}
+
+// Information about each individual server, whose group versions needs to be summarized.
+type FederatedServer struct {
```

Comment:
I can rename to Server if you think its better. ServerGroup doesnt sound right since this can have multiple groups.
FederatedServer is not that long.